chainguard-dev/edu
# remove build-tools section from edu

## Type of change

Content removal.

### What should this PR do?

Remove this section and the `apko` and `melange` subsections: https://edu.chainguard.dev/open-source/build-tools/

### Why are we making this change?

These docs can be moved to the respective GH repos, where they can get updated as changes are made to these tools.

### What are the acceptance criteria? 

No broken links, no broken update automation.

### How should this PR be tested?

Check the preview that I removed things right, and removed enough.

I'm not sure how to test how this affects update automation though.
